airflow/operators/docker_operator.py

```diff
@@ -47,6 +47,10 @@ class DockerOperator(BaseOperator):
     :param api_version: Remote API version. Set to ``auto`` to automatically
         detect the server's version.
     :type api_version: str
+    :type auto_remove: bool
```

Comment:
Please switch the order. First `:param`, then `:type`.

Reply:
You're right. I'm going to modify it soon.